README.md

```diff
@@ -1706,7 +1703,6 @@ Usage:
        json, returning `true` if there is a match, `false` otherwise.
   - returns
     - attributes ( including relationships ) to include in response json
-- Need to import `run` from `@ember/runloop` and wrap tests using `mockCreate` with: `run(function() { 'your test' })`
```

Comment:
As far as I can tell, this isn't needed in tests anymore. I suspect testing has improved a lot since this was written